src/docs/asciidoc/integration.adoc

```diff
@@ -6680,8 +6680,35 @@ in combination with a custom pointcut.
 [[scheduling-annotation-support-qualification]]
 ==== Executor Qualification with `@Async`
 
-By default, when specifying `@Async` on a method, the executor that is used is the
-one supplied to the "`annotation-driven`" element, as described earlier. However, you can use the `value`
+As seen previously, you can enable the `@Async` annotation by adding `@EnableAsync` to any of your
+`@Configuration` classes:
+
+====
+[source,java,indent=0]
+[subs="verbatim,quotes"]
+----
```

Comment:
Explaining how to configure `@EnableAsync` in a section named "Executor Qualification with `@Async`" looks odd to me. This is also a copy/paste of the javadoc of `@EnableAsync` so I am not too keen to duplicate that.

Can we take a step back and can you describe why you'd want this to be added here?

Reply:
Sure. The main purpose of this PR is as described on https://jira.spring.io/projects/SPR/issues/SPR-17458 to make more clear where the _XML-based 'annotation-driven' element_ default considered executor is coming from. [As also commented here](https://jira.spring.io/browse/SPR-17458?focusedCommentId=163124&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-163124) unless there is a way to link directly to the referred `task:annotation-driven `XML definition (which in my opinion is a better approach), duplicating it in this section is the only other way. 

In particular the initial section you are quoting here is indeed not related to the executor usage but I've included it because that's how I've interpreted the last section of the last phrase of the description of the ticket: _'and make sure it refers to annotations'_. Unless I got it wrong of course - maybe it means an `@Bean` Java Config definition of an Executor?